drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenterTest.java

```diff
@@ -1374,13 +1391,17 @@ public void onDeleteSelectedCellsWithSelectionWithBooleanColumn() {
         verify(synchronizer,
                never()).deleteCell(any(GridData.Range.class),
                                    any(Integer.class));
+        checkDTSelectionsChangedEventFired(1);
     }
 
     @Test
     public void onDeleteSelectedCellsWithSelectionsWithBooleanColumn() {
-        final AttributeCol52 column = new AttributeCol52() {{
-            setAttribute(RuleAttributeWidget.ENABLED_ATTR);
-        }};
+        final AttributeCol52 column = new AttributeCol52() {
```

Comment:
Is this Eclipse's formatting of static initialiser blocks (in anonymous sub-classes!)